packages/graphql/.eslintignore

```diff
@@ -0,0 +1 @@
+**/tsconfig.json
```

Comment:
why don't we want to link the tsconfig?

Reply:
I added these ignore files because there's a plugin that is already skipping these files and when it does it logs a warning. Wanted to clean those up

```
/root/cypress/packages/graphql/tsconfig.json
  0:0  warning  [@cypress/eslint-plugin-json]: Skipping file due to "json/json-with-comments-files" setting: 
  ["**/tsconfig.json", ".vscode/**"]
To remove this warning add "tsconfig.json" to your `.eslintignore` file
```